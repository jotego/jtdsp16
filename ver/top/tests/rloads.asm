# Test register loads
move r0 = 0xFFFF
move r0 = 0
move r1 = 0x11
move r2 = 0x22
move r3 = 0x33
move r1 = 0x1111
move r2 = 0x2222
move r3 = 0x3333

# j, k, rb, re
move j = 0x4004
move k = 0x5005
move rb= 0x6006
move re= 0x7007

# pt, pr, i
pt = 0xbeef
pr = 0xcade
i  = 0xae00

# DAU
x=0x6532
yl=0x2312
auc=0x6
#psw=0xdeed
c0=0x10
c1=0x11
c2=0x12

end:
goto end
