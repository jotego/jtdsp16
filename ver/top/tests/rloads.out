r0=0x0000
r1=0x1111
r2=0x2222
r3=0x3333
rb=0x6006
re=0x7007
j =0x4004
k =0x5005
pr=0xcade
pt=0xbeef
i =0x0e00
a0=0x000000000
a1=0x000000000
x=0x6532
y=0x00002312
p=0x00000000
c0=0x0010
c1=0x0011
c2=0x0012
auc=0x0006
psw=0x0000
